<h2>While Loop</h2>
    
Buatlah sebuah code while loop sebagai berikut :
- Buatlah suatu program yang merima satu input nilai numerik
- Buat sebuah while-loop dengan melakukan comparasi terhadap inputan tersebut, dimana looping/iterasi terus berjalan apabila nilai inputan tidak sama dengan 0
- Dalam setiap iterasi update nilai variable inputan itu, jika input lebih dari 0, maka kurangi variabel tersebut sebesar 1, selain itu tambahkan 1
- print nilai variable input dalam setiap awal iterasi

In [4]:
x = 10
while x != 0:
    if x > 0:
        print(x)
        x = x - 1
    else:
        x = x + 1

10
9
8
7
6
5
4
3
2
1


Expected Output:

10

9

8

7

6

5

4

3

2

1

<h2>For Loop</h2>
    
Lengkapi kode untuk menghasilkan suatu output yang di harapkan:

- Buatlah sebuah loop dengan mengiterasi sebuah objek list kemudian di kalikan dengan index dari list tersebut dan print hasilnya

In [5]:
obj_list= [1, 16, 11, 10, 5]

# lengkapi code di bawah
for x in obj_list :
    print(obj_list.index(x) * x)

0
16
22
30
20


Expected Output:

0

16

22

30

20


<h2>Loop Dictionary</h2>
    
Lengkapi kode untuk menghasilkan suatu output yang di harapkan

In [6]:
country = { 
           'spain': { 'capital':'madrid', 'population':46.77 },
           'france': { 'capital':'paris', 'population':66.03 },
           'germany': { 'capital':'berlin', 'population':80.62 },
           'norway': { 'capital':'oslo', 'population':5.084 },
           'indonesia' : {'capital':'jakarta', 'population':250}
         }

for x, y in country.items():
    print('Ibukota ' + x + ' adalah ' + y['capital'])

Ibukota spain adalah madrid
Ibukota france adalah paris
Ibukota germany adalah berlin
Ibukota norway adalah oslo
Ibukota indonesia adalah jakarta


Expected Output:
    
Ibukota spain adalah madrid

Ibukota france adalah paris

Ibukota germany adalah berlin

Ibukota norway adalah oslo

Ibukota indonesia adalah jakarta

<h2>Exercise 1</h2>

buatlah perulangan untuk menghitung jumlah dari kuadrat 10 angka bilangan berikutnya dari variabel start (10+11+12+...)

In [7]:
#perulangan untuk menjumlah kuadrat
start = 10
sum_number = 0

for start in range(10,20):
  sum_number = (start ** 2) + sum_number   
print("Jumlah kuadrat: {}".format(sum_number))

Jumlah kuadrat: 2185


output yang diminta:

Jumlah Kuadrat: 2185

nilai tersebut didapatkan dari 10 x 10 + 11 x 11 + ..... + 19 x 19 

<h2>Exercise 2</h2>

buatlah program interaktif yang akan menampilkan angka dalam format segitiga (baris pertama hanya diisi oleh angka 1, baris kedua diisi oleh angka 1,2, baris ketiga 1,2,3, dst)

In [8]:
#mengecek apakah elemen dari array satuan atau tidak
n = int(input('Enter the number of rows = '))
    
#untuk membuat baris kosong, silakan menggunakan print('\n')
#untuk menampilkan nilai ke samping tanpa membuat garis baru, bisa menggunakan print(x, end=' ')

for x in range(1, n + 1):
    for y in range(1, x + 1):
        print(y, end=' ')
    print('\n')

Enter the number of rows = 5
1 

1 2 

1 2 3 

1 2 3 4 

1 2 3 4 5 



Masukkan Jumlah Baris: 5

1

1 2

1 2 3

1 2 3 4

1 2 3 4 5
